qutebrowser/mainwindow/prompt.py

```diff
@@ -721,6 +732,7 @@ def _allowed_commands(self):
             ('prompt-accept', 'Accept'),
             ('leave-mode', 'Abort'),
             ('prompt-open-download', "Open download"),
+            ('prompt-yank', "Yank URLs in prompts"),
```

Comment:
This is about the help text being shown in prompts. Something like "Yank URL" would be more appropriate (if it's always an URL).